tests/commands/test_rele_cli.py

```diff
@@ -0,0 +1,10 @@
+from rele.__main__ import run_worker
+
+
+class TestReleCli:
+    def test_rele_cli_run(self, mock_worker):
+        run_worker("tests.settings", ["sample_pypi_package.subs"])
```

Comment:
What about add test for the validation of the module.
Could be easy as adding "sample_pypi_package.no_subs" in this happy case or add anew test like:
```
    def test_ignores_non_valid_third_party_subs(self, mock_worker):
        run_worker("tests.settings", ["sample_pypi_package.no_subs"])

        mock_worker.assert_called()
```